projects/plugins/jetpack/extensions/blocks/paywall/edit.js

```diff
@@ -1,19 +1,15 @@
-import { BlockIcon } from '@wordpress/block-editor';
-import { Placeholder } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import './editor.scss';
-import icon from './icon';
 
 function PaywallEdit( { className } ) {
+	const text = __( 'Paywall', 'jetpack' );
```

Comment:
We'll probably want to have something that is instructive, e.g. "Paid content below" or "Divides paid and free content", but you can keep "Paywall" until we have final copy ready to send for translating.